src/engine/QueryExecutionTree.h

```diff
@@ -138,6 +133,10 @@ class QueryExecutionTree {
       const vector<string>& selectVars, size_t limit, size_t offset,
       shared_ptr<const ResultTable> preComputedResult = nullptr) const;
 
+  template <ExportSubFormat format>
+  ad_utility::stream_generator::stream_generator writeTable(
+      const vector<string>& selectVars, size_t limit, size_t offset) const;
+
```

Comment:
I just realized I dropped the default parameters by accident. Probably not an issue, noteworthy nevertheless..

It's also open for debate if the new single function should be called `generateResults` or `writeTable`

Reply:
`generateResults` is my strong favourite, it would also be more consistent.
And if the default parameters are used nowhere, then please drop them, they only increase accidental errors.